homeassistant/components/efesto/climate.py

```diff
@@ -0,0 +1,290 @@
+"""Support for Efesto heating devices."""
+import logging
+
+from efestoclient import (
+    EfestoClient,
+    Error,
```

Comment:
I would suggest that you rename this error since its name is too arbitrary e.g. 
```python
Error as EfestoError
```

Reply:
I agree, changed now.